RebelAllianceBank/Users/Customer.cs

```diff
@@ -309,21 +321,17 @@ public void TakeLoan()
                     }
 
                     Console.WriteLine("\nVälj vilket konto du vill sätta in pengarna på:\n");
-                    for (int i = 0; i < _bankAccounts.Count; i++)
-                    {
-                        Console.WriteLine($"{i + 1}. {_bankAccounts[i].AccountName}");
-                    }
-                    if (!int.TryParse(Console.ReadLine(), out choosenAccountIndex) || choosenAccountIndex <= 0 || choosenAccountIndex > _bankAccounts.Count)
-                    {
-                        Console.WriteLine("Felaktigt val.");
-                        continue;
-                    }
+
+                    // int choosenAccountIndex = MarkdownUtils.HighLightChoiceWithMarkdown(false, ["id", "Konto Namn", "Saldo", "Valuta"], PopulateAccountDetails(_bankAccounts), option => [option]);
+                    int choosenAccountIndex = new SelectOneOrMore(["id", "Konto Namn", "Saldo", "Valuta"], PopulateAccountDetails(_bankAccounts)).Show()[0];
```

Comment:
Finns det en specifik anledning till att du använder SelectOneOrMore metoden istället för HighLightChoiceWithMarkdown? Tänker att den metoden skulle funka bättre här, men är inte säker.

Reply:
It was because @sidstedt Method had an error when the user has only one account.